### PR TITLE
We don't want to emit warnings when parsing the data before updating …

### PR DESCRIPTION
**Pull request by dmarcos**
…the schema

### CODE REVIEW

File: src/core/component.js [hunk 1]
```diff
@@ -110,12 +110,13 @@ Component.prototype = {
    * If component is single-property, then parses the single property value.
    *
    * @param {string} value - HTML attribute value.
+   * @param {boolean} silent - Suppress warning messages.
```
Comment (ngokevin) [inline src/core/component.js]: --> ```Surpress```

File: src/core/component.js [hunk 2]
```diff
@@ -248,8 +249,15 @@ module.exports.registerComponent = function (name, definition) {
  * 3. Attribute data.
  *
  * Finally coerce the data to the types of the defaults.
+ *
+ * @param {object} el - Element we're build data from.
```
Comment (ngokevin) [inline src/core/component.js]: --> ```Element to build data from.```

File: src/core/component.js (hunk 1 above)
Comment (dmarcos) [inline src/core/component.js]: Did you mean `suppress`? http://dictionary.reference.com/browse/suppress

Comment (ngokevin) [inline src/core/component.js]: I think here would be a good place to explain why we have this flag.

```Generally want to use while swapping the schema like with material shaders.```

**Comment by ngokevin** [comment]
r+wc wfm!

File: src/core/component.js (hunk 1 above)
Comment (ngokevin) [inline src/core/component.js]: Haha, yeah, suppress, oops.

**Pull request merged**